auction-house/program/src/auctioneer/update.rs

```diff
@@ -0,0 +1,67 @@
+use anchor_lang::prelude::*;
+
+use crate::{constants::*, errors::AuctionHouseError, AuctionHouse, Auctioneer, AuthorityScope};
+
+#[derive(Accounts)]
+pub struct UpdateAuctioneer<'info> {
+    // Auction House instance PDA account.
+    #[account(
+        mut,
+        seeds = [
+            PREFIX.as_bytes(),
+            auction_house.creator.as_ref(),
+            auction_house.treasury_mint.as_ref()
+        ], 
+        bump=auction_house.bump, 
+        has_one=authority
+    )]
+    pub auction_house: Account<'info, AuctionHouse>,
+
+    #[account(mut)]
+    pub authority: Signer<'info>,
+
+    /// CHECK: The auction house authority can set this to whatever external address they wish.
+    /// The auctioneer authority - typically a PDA of the program running this auction.
```

Comment:
I added nit comments about this 1000 times. Close them out if this is supposed to be "auction"

The code looks good. Thanks!